src/App.tsx

```diff
@@ -30,7 +29,8 @@ import {
   ChatClient,
   useMuminstApi,
 } from 'features/api/useMuminstApi';
-import { fuzzyFilter } from 'features/search/useFuzzyFilter/fuzzyFilter';
```

Comment:
delete the code and also remove `fuzzaldrin` from the dependency list

Reply:
done